src/device-registry/utils/create-health-tips.js

```diff
@@ -24,12 +25,17 @@ const createHealthTips = {
       const limit = parseInt(request.query.limit, 0);
       const skip = parseInt(request.query.skip, 0);
       const filter = generateFilter.tips(request);
+      const language = request.query.language;
 
       const responseFromListHealthTips = await HealthTipModel(tenant).list({
         filter,
         limit,
         skip,
       });
+      if (language != null) {
```

Comment:
Hi @BenjaminSsempala 
Please consider using `!==` comparison operator for benefits you can search up online. Thanks!

Reply:
Thanks @Baalmart